package.json

```diff
@@ -16,6 +16,6 @@
     "tree-sitter-cli": "^0.15.14"
   },
   "scripts": {
-    "test": "tree-sitter test"
+    "test": "tree-sitter generate && tree-sitter test"
```

Comment:
I don't think we want to regenerate every time we run tests, because regenerating and recompiling takes much longer. Could you back this part out?

Reply:
Yes that was a mistake! I use it like that locally to verify my changes to the grammar.